src/cheevos/Cheevos.cpp

```diff
@@ -91,6 +98,102 @@ void CCheevos::EvaluateRichPresence(char* evaluation, size_t size)
   rc_evaluate_richpresence(m_richPresence, evaluation, size, PeekInternal, this, NULL);
 }
 
+void CCheevos::ActivateAchievement(unsigned cheevo_id, const char* memaddr) //1
+{
+  int res = rc_runtime_activate_achievement(&m_runtime, cheevo_id, memaddr, NULL, 0);
```

Comment:
you could make this method to return bool and do something like the other methods
```suggestion
return rc_runtime_activate_achievement(&m_runtime, cheevo_id, memaddr, NULL, 0) == 0;
```